internal/store/device.go

```diff
@@ -26,6 +26,7 @@ type Device interface {
 	Create(ctx context.Context, orgId uuid.UUID, device *api.Device, callback DeviceStoreCallback) (*api.Device, error)
 	Update(ctx context.Context, orgId uuid.UUID, device *api.Device, fieldsToUnset []string, fromAPI bool, callback DeviceStoreCallback) (*api.Device, error)
 	List(ctx context.Context, orgId uuid.UUID, listParams ListParams) (*api.DeviceList, error)
+	Summary(ctx context.Context, orgId uuid.UUID, listParams ListParams) (*api.DevicesSummary, error)
```

Comment:
@avishayt @fzdarsky, I have decided to keep the Device Summary for components that might use it.
The API of ListDevices will handle the case when "SummaryOnly" is specified and call the summary accordingly.
No new API is presented in this case.